fix: naive and aware datetimes comparison

Faraz32123 opened the pull request:
In this PR:
- fixes `TypeError: can't compare offset-naive and offset-aware datetimes` error by making datetime.now(), aware datetime.
- bump version

ziafazal commented on h5pxblock/h5pxblock.py:
```diff
@@ -445,7 +445,8 @@ def is_past_due(self):
             return False
-        return datetime.now() > self.due
+        due_tz = self.due.tzinfo
+        return datetime.now(due_tz) > self.due
```
I'm not sure why you need a single use variable `due_tz ` here. Can we do it like this instead?
```suggestion
        return datetime.now(self.due.tzinfo) > self.due
```

Commit:
fix: naive and aware datetimes comparison

this commit:
- fixes `TypeError: can't compare offset-naive and offset-aware datetimes` error by making datetime.now(), aware datetime.
- bump version

ziafazal reviewed (approved):
👍
